README.md

```diff
@@ -313,6 +313,80 @@ Some Serilog packages require a reference to a logger configuration object. The
 },
 ```
 
+### Destructuring
+
+Destructuring means extracting pieces of information from an object and create properties with values; Serilog offers the [@ destructuring operator](https://github.com/serilog/serilog/wiki/Structured-Data#preserving-object-structure). In case there is a need to customize the way log events are serialized (e.g., hide property values or replace them with something else), one can define several destructuring policies, like this:
+
+```yaml
+"Destructure": [
+  {
+    "Name": "With",
+    "Args": {
+      "policy": "MyFirstNamespace.FirstDestructuringPolicy, MyFirstAssembly"
+    }
+  },
+  {
+    "Name": "With",
+    "Args": {
+      "policy": "policy": "MySecondNamespace.SecondDestructuringPolicy, MySecondAssembly"
+    }
+  },
+   {
+    "Name": "With",
+    "Args": {
+      "policy": "policy": "MyThirdNamespace.ThirdDestructuringPolicy, MyThirdAssembly"
+    }
+  },
+],
+```
+
+This is how the first destructuring policy would look like:
+
+```csharp
+namespace MyFirstNamespace
+{
+  public class MyDto
+  {
```

Comment:
Not sure how much of the C# world is on board with records yet, but:

```csharp
public record MyDto(int Id, int Name);
```

would cut a few lines?